recipe/conda_build_config.yaml

```diff
@@ -14,3 +14,8 @@ cxx_compiler:                                   # [win]
 
 nodejs:
 - '14'
+
+MACOSX_DEPLOYMENT_TARGET: # [osx and x86_64]
+  - "10.15"               # [osx and x86_64]
```

Comment:
Could it be that this hasn't been rerendered yet? The CI files still contain 10.11. Also, we should as a dun-constraint on `__osx` if we raise the minimum target

Reply:
Adding this to the meta.yaml will be part of a follow-on macos-specific PR
```
requirements:
  run:
    - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.15") }}  # [osx and x86_64]
```